docs/connect_bolt_pe.md

```diff
@@ -123,7 +123,7 @@ config:
   transport: pcp
   pcp:
     cacert: "certs/cert.pem"
-    service-url: "https://master.example.com"
+    service-url: "https://master.example.com:8150"
```

Comment:
The example should probably by 8143, since that's what the orch port is by default.